src/app/components/test/lectura/lectura.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, OnInit } from '@angular/core';
+import { NgForm } from '@angular/forms';
+import { Router } from '@angular/router';
+import { CommonService } from 'src/app/services/common/common.service';
+import { LecturaService } from 'src/app/services/test/lectura.service';
+import { GameCategoryRequest } from '../../commons/models/commons/GameCategoryRequest';
+import { PatientTaskAnswersRequestList } from '../../commons/models/commons/PatientTaskAnswersRequestList';
+
+@Component({
+  selector: 'app-lectura',
+  templateUrl: './lectura.component.html',
+  styleUrls: ['./lectura.component.scss'],
+})
+export class LecturaComponent implements OnInit {
+  respuesta: string[] = [];
+  retorno: boolean;
+  cargando = false;
+  errorCode = false;
+  gameId: number;
+  category: string;
+  taskId: number;
+  descripcion: string;
+  url = 'https://prod-mobius-mind-api.herokuapp.com/';
+  img = 'images/';
+  imgName: string;
+  imagen: string;
+  lecturaRequest: GameCategoryRequest = null;
+
+  constructor(public commonService: CommonService, private router: Router, public lecturaServ: LecturaService) { }
+
+  ngOnInit() {
+    this.obtenerDatos();
+  }
+
+  obtenerDatos(){
+
+    this.lecturaRequest = new GameCategoryRequest();
+    this.lecturaRequest.patientTaskAnswersRequestList = new Array<PatientTaskAnswersRequestList<string>>();
+
+    this.lecturaServ.traerDatos().subscribe((resp: any) => {
+      this.descripcion = resp.tasks[0].description;
+      this.imgName = resp.resources[0].fileName;
+      console.log(this.descripcion);
+      this.lecturaRequest.gameId = resp.id;
+      this.lecturaRequest.category = resp.category;
+      console.log(this.lecturaRequest.gameId);
+      console.log(this.lecturaRequest.category);
+      this.taskId = resp.tasks[0].id;
+      console.log(this.taskId);
+      console.log(resp);
+      console.log(this.lecturaRequest);
+      this.imagen = `${this.url}${this.img}${this.imgName}`;
```

Comment:
porque estas llamando al service desde aca? si lo tenes que llamar debe pasarse a otro metodo

Reply:
osea el service es el q se comunica con el backend y me trae el json, en el metodo obtenerDatos() uso el servicio justamente para eso, sino explicame donde deberia ir, lo quite de ngOnInit() por lo que me dijiste ayer. La funcion del metodo obtenerDatos() es justamente esa, agarrar el json y pasar la informacion q necesito a variables q pueda utilizar.

Reply:
this.imagen = `${this.url}${this.img}${this.imgName}`; --> esto que hace en el componente
Osea que pases a un metodo que vaya al service que busque esto this.imagen = `${this.url}${this.img}${this.imgName}`; y llamarlo desde el componente ts a este service . se entiende?

Reply:
fer te entiendo pero no le veo el sentido a hacer un paso extra, me explico: 
${this.url} tiene la url q saque de enviroment.prod.url, 
${this.img} tiene "images/" q es la carpeta donde eze ahora guarda las imágenes y 
${this.imgName} tiene el nombre del archivo que acabo de traer del servicio, osea tengo todo lo necesario en el componente para formar la url y del servicio ya pedí los datos.

vos lo que queres es que haga un método en lecturaService para que haga todo ese proceso? mi idea era cuando termine el sprint UNIFICAR todos estos services que fui creando en uno solo, porq el método traerDatos y enviarDatos de los services se repite en todos lados, con pasarle una variable indicando a q test pertenece podría unificar todos los test en uno, pero ya el tema imágenes es distinto, no todos los test usan imágenes y meter un método así en un services general nose como quedaría. 

Te dejo el aviso por eso desde que me indicaste que usando GameCategoryRequest puedo crear el mensaje pensé en unificar todos los servicios en uno pero hacerlo hoy a jueves tan cerca del sprint no me da confianza, luego del sábado hago eso.

Reply:
Buena respuesta, despues lo vemos junto a eso...te salvaste